yarn.lock

```diff
@@ -3725,10 +3725,10 @@ uuid@^3.3.2:
   resolved "https://registry.yarnpkg.com/uuid/-/uuid-3.3.2.tgz#1b4af4955eb3077c501c23872fc6513811587131"
   integrity sha512-yXJmeNaw3DnnKAOKJE51sL/ZaYfWJRl1pK9dr19YFCu0ObS231AB1/LbqTKRAQ5kw8A90rA6fr4riOUpTZvQZA==
 
-uuid@^8.3.2:
-  version "8.3.2"
-  resolved "https://registry.yarnpkg.com/uuid/-/uuid-8.3.2.tgz#80d5b5ced271bb9af6c445f21a1a04c606cefbe2"
-  integrity sha512-+NYs2QeMWy+GWFOEm9xnn6HCDp0l7QBD7ml8zLUmJ+93Q5NF0NocErnwkTkXVFNiX3/fpC6afS8Dhb/gz7R7eg==
+uuid@^9.0.0:
+  version "9.0.0"
+  resolved "https://na.artifactory.swg-devops.com/artifactory/api/npm/appconnect-npm/uuid/-/uuid-9.0.0.tgz?dl=https%3A%2F%2Fregistry.npmjs.org%2Fuuid%2F-%2Fuuid-9.0.0.tgz#592f550650024a38ceb0c562f2f6aa435761efb5"
```

Comment:
Hm, I would expect the url to be from be `registry.yarnpkg.com`, that's what I get when I do it locally as well.

It feels like I'm asking you to do busywork though, if you don't mind I could close this PR and do the upgrade myself (with references though, I don't intend to steal the credit).

Reply:
Oh shoot, that's what I get for running `yarn install` on my work machine... we have our own internal npm registry! By all means close this PR and handle this yourself. Thank you. And well spotted!